### PR TITLE
Add option to noindex the post for Google News

### LINKED ISSUE #138: [Feature Request] Add Support For Googlebot-News 'noindex' Meta
Google can index posts in Google News even though they are not listed in the News Sitemap. Customer requested that we add a Google News specific noindex option as specified [here](https://support.google.com/news/publisher/answer/93977?hl=en).

`<meta name="Googlebot-News" content="noindex, nofollow">`

### PR DESCRIPTION
**Pull request by andizer**
Fixes #138

### CODE REVIEW

File: classes/class-meta-box.php [hunk 1]
```diff
@@ -81,6 +81,17 @@ public function get_meta_boxes( $post_type = 'post' ) {
 				'title'       => __( "Editors' Pick", 'wordpress-seo-news' ),
 				'description' => __( "Editors' Picks enables you to provide up to five links to original news content you believe represents your organization’s best journalistic work at any given moment, and potentially have it displayed on the Google News homepage or select section pages.", 'wordpress-seo-news' ),
 			),
+			'newssitemap-robots-index' => array(
+				'type'          => 'radio',
+				'default_value' => '0', // = index.
```
Comment (jcomack) [inline classes/class-meta-box.php]: Consider adding a better piece of description here. Perhaps something along the lines of `This refers to the possible options listed in 'options'`.

File: classes/class-head.php [hunk 2]
```diff
@@ -96,4 +97,17 @@ private function display_standout() {
 			}
 		}
 	}
+
+	/**
+	 * Shows the meta-tag with noindex when there is decided to exclude the post from Google News.
```
Comment (jcomack) [inline classes/class-head.php]: This isn't grammatically correct. Consider changing it to `[...] when it has been decided [...]`.

**Comment by jcomack** [comment]
CR done 🚧

File: classes/class-head.php (hunk 2 above)
Comment (andizer) [inline classes/class-head.php]: Thanks 👍

File: classes/class-meta-box.php [hunk 3]
```diff
@@ -81,6 +81,17 @@ public function get_meta_boxes( $post_type = 'post' ) {
...
+			'newssitemap-robots-index' => array(
+				'type'          => 'radio',
+				'default_value' => '0', // = The default value will be 'index'; See the list of options.
+				'std'           => '',
+				'options'       => array(
+					'0' => __( 'index', 'wordpress-seo-news' ),
+					'1' => __( 'noindex', 'wordpress-seo-news' ),
+				),
+				'title'         => __( 'Googlebot-News index', 'wordpress-seo-news' ),
+				'description'   => __( 'With noindex you will prevent this article from appearing in Google News.', 'wordpress-seo-news' ),
```
Comment (jcomack) [inline classes/class-meta-box.php]: I'd alter this to something like `Using noindex allows you to prevent articles from appearing in Google News`.

**Comment by moorscode** [comment]
CR + Acceptance done

**Pull request merged**